docs/src/main/sphinx/security/kerberos.rst

```diff
@@ -50,7 +50,7 @@ Configuration for TLS
 ^^^^^^^^^^^^^^^^^^^^^
 
 When using Kerberos authentication, access to the Trino coordinator must be
-through :doc:`TLS and HTTPS </security/tls>`.
+through :doc:`HTTPS and TLS </security/tls>`.
```

Comment:
I've tried to standardize on TLS first. There's even a Jira for that. 

TLS is the actual protocol used that we want people to get used to seeing.